src/main/java/at/hannibal2/skyhanni/features/fishing/trophy/OdgerWaypoint.kt

```diff
@@ -37,7 +38,7 @@ class OdgerWaypoint {
 
     private fun isLavaFishingRod(): Boolean {
         val heldItem = InventoryUtils.getItemInHand() ?: return false
-        val isRod = heldItem.name?.contains("Rod") ?: return false
+        val isRod = heldItem.getItemCategoryOrNull() == ItemCategory.FISHING_ROD
```

Comment:
sorry,i missed the thing with the fishing weapon
```suggestion
        val isRod = heldItem.getItemCategoryOrNull() == ItemCategory.FISHING_ROD || heldItem.getItemCategoryOrNull() == ItemCategory.FISHING_WEAPON
```

Reply:
Should be cleaner (not tested):
```suggestion
        val isRod = heldItem.getItemCategoryOrNull() in arrayOf(ItemCategory.FISHING_ROD, ItemCategory.FISHING_WEAPON)
```

Reply:
even better
```suggestion
        val isRod = heldItem.getItemCategoryOrNull() in setOf(ItemCategory.FISHING_ROD, ItemCategory.FISHING_WEAPON)
```

Reply:
But maybe move the function into the FishingAPI.

Reply:
Yeah, should just fix the function there and then use that here.